src/mixins/index.js

```diff
@@ -147,7 +147,7 @@ const methods = {
         maximumFractionDigits: decimals,
       })
 
-      return appendCurrency ? `${value} ${store.getters['network/symbol']}` : value
+      return appendCurrency ? `${value} ${ store.getters['network/symbol'] || "" }` : value
```

Comment:
```suggestion
      return appendCurrency ? `${value} ${
          store.getters['network/symbol'] ||
          store.getters['network/defaults'].symbol ||
          ""
        }` : value
```

This would be a better approach imho. You'd need to add the symbol of the respective network to the network files, e.g. https://github.com/ArkEcosystem/explorer/blob/master/networks/mainnet.json#L43